openpype/tools/utils/lib.py

```diff
@@ -149,7 +169,9 @@ def get_task_icon():
 
     Icon should be defined by task type which is stored on project.
     """
-    return get_qta_icon_by_name_and_color("fa.male", style.colors.default)
+    return get_qta_icon_by_name_and_color(
+        "fa.male", get_default_entity_icon_color()
+    )
```

Comment:
This is hardcoded to a single icon, unlike what the docstring describes. Avalon allowed to have a different icon per task name (which was basically it's type). Did we break that logic in OpenPype somewhere? Shouldn't this return an icon fitting with that particular task type?